### PR TITLE
Deprecation notice

### PR DESCRIPTION
**Pull request by esreli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# Deprecation Notice
+
+Esri ceases providing support for the Ecological Marine Unit Explorer app for Android and the repository is archived as public read-only. You are free to continue to use the source code operating under the existing [license](./LICENSE).
```
Comment (jf990) [inline README.md]: suggested copy change:

The Ecological Marine Unit Explorer app for Android is no longer supported and the repository is archived as public read-only. You are free to continue to use the source code under the existing [license](./LICENSE).

File: README.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+## What do I need to do?
+
+No action is needed.
```
Comment (jf990) [inline README.md]: Do we need to say this? Seems superfluous.

Comment (esreli) [reply README.md]: Okay, ill scratch it.

File: README.md (hunk 1 above)
Comment (esreli) [reply README.md]: I like this copy better. Swapped.

**Pull request merged**